### PR TITLE
dts: radxa cm3 rpi cm4 io: add pcf85063 rtc

### PR DESCRIPTION
**Pull request by vamrs-feng**

### CODE REVIEW

File: arch/arm64/configs/rockchip_linux_defconfig [hunk 1]
```diff
@@ -1305,3 +1305,4 @@ CONFIG_RCU_CPU_STALL_TIMEOUT=60
 CONFIG_FUNCTION_TRACER=y
 CONFIG_BLK_DEV_IO_TRACE=y
 CONFIG_LKDTM=y
+CONFIG_RTC_DRV_PCF85063=y
```
Comment (RadxaYuntian) [inline arch/arm64/configs/rockchip_linux_defconfig]: 结尾空行

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/rockchip/rk3566-radxa-cm3-rpi-cm4-io.dts [hunk 2]
```diff
@@ -133,6 +133,14 @@
 			pwm-min = <0>;
 		};
 	};
+
+	pcf85063: pcf85063@51 {
+		compatible = "nxp,pcf85063";
+		status = "okay";
+		reg = <0x51>;
+		#clock-cells = <0>;
```
Comment (RadxaYuntian) [inline arch/arm64/boot/dts/rockchip/rk3566-radxa-cm3-rpi-cm4-io.dts]:
```suggestion
```
文档里面这个应该是子节点的属性：https://www.kernel.org/doc/Documentation/devicetree/bindings/rtc/nxp%2Cpcf85063.txt

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: arch/arm64/configs/rockchip_linux_defconfig [hunk 3]
```diff
@@ -1305,3 +1305,5 @@ CONFIG_RCU_CPU_STALL_TIMEOUT=60
...
+
```
Comment (RadxaYuntian) [inline arch/arm64/configs/rockchip_linux_defconfig]: 你这是多了个空行吧？文件结尾应该只有一个空行

**Review by RadxaYuntian — APPROVED** [review]

**Comment by RadxaStephen** [comment]
Please modify rockchip_linux_defconfig via make menuconfig.

**Comment by RadxaStephen** [comment]
Usually we have at least three kinds of commits, dts, driver, defconfig.
We should have separate commits for dts, driver, defconfig.

Here, please provide one commit for rk3566-radxa-cm3-rpi-cm4-io.dts and another one commit for rockchip_linux_defconfig.

**Pull request merged**